plugins/org.eclipse.sirius.diagram.ui/src-diag/org/eclipse/sirius/diagram/ui/business/internal/view/LayoutData.java

```diff
@@ -215,7 +215,7 @@ public List<LayoutData> getChildren() {
     }
 
     /**
-     * Search recursively in in all the LayoutData is there is one which have
```

Comment:
You missed "Search recursively **in in** [...]"

Reply:
Done (plus another "in in" comment)